app/assets/stylesheets/hyrax/_positioning.scss

```diff
@@ -16,12 +16,13 @@
   padding: 0.5em 0;
 }
 
-.workflow-affix {
-  bottom: 0;
+.panel-workflow {
   background: #FFF;
-  z-index: 2;
+  border-color: #e0e0e0;
+  bottom: 0;
   left: 0;
-  right: 0;
-  max-width: 1140px;
   margin: 0 auto;
+  max-width: 1140px;
+  right: 0;
+  z-index: 2;
```

Comment:
Do we still need `bottom`, `z-index` and `right` since it's no longer fixed?

Reply:
Probably not. I will remove and test.